types/node/http.d.ts

```diff
@@ -172,6 +182,21 @@ declare module "http" {
         setTimeout(timeout: number, callback?: () => void): this;
         setNoDelay(noDelay?: boolean): void;
         setSocketKeepAlive(enable?: boolean, initialDelay?: number): void;
+
+        addListener(event: 'information', listener: (info: InformationEvent) => void): this;
+        addListener(event: string | symbol, listener: (...args: any[]) => void): this;
```

Comment:
Doesn't this hide the `addListener`,... variants from base class? sure, there is the generic variant so nothing is broken but now only the `information` event is typed.
On the other hand copy pasting all the other definitions is also not nice...

Reply:
Well, that's a problem, not only in this place but already existing ones as well.
I will use declaration merging instead.

Reply:
Which.. apparently doesn't work anymore, this is going to become annoying.

Reply:
With newer typescript versions this could be improved to define lists of types and reuse them at several places... but we have to stick on 2.0/2.1...